### ISSUE TITLE
Resources not find on developer console

### ISSUE THREAD

**Issue by paulmorriss** [ethereum/meteor-dapp-wallet#49]
I'm running on Windows 7 64 bit. (Behind a firewall, so no peers, but that's probably not relevant.)
When I go into the developer console there are three messages:
Failed to load resource: net::ERR_FILE_NOT_FOUND
The file are:
file:///C:/sounds/bip.mp3
file:///sockjs/info?cb=wbua9fc8de 
file:///C:/(install location)/resources/app.asar/interface/wallet/packages/ethereum_dapp-styles/fonts/SourceSansPro-ExtraLightIt.otf

**Comment by frozeman** [comment]
This has todo with the file protocol and how electron loads it basically gives you `file:///` without any path and therefore all relative paths won't work.

Its not a big issue, and the sound will be fixed in the next release (Beta3).

**Comment by alexvandesande** [comment]
Only not found error that shows on the latest beta is
`file:///sockjs/info?cb=7bgj2co31l`

Doesn't seem like a real issue.

**Comment by pipilangstrumpf** [comment]
Hi Guys!

I have a similiar problem. I couldnt mine Ether in the Testnet. So I opened the Wallet UI and got the Error message:
![ethereum error](https://cloud.githubusercontent.com/assets/17416752/13233646/e8a3869a-d9b4-11e5-8ac0-560d0a39dfb1.PNG)

Later i tried it again and the message changed a little bit. In the testnet and in the main network.

![ethereum error2](https://cloud.githubusercontent.com/assets/17416752/13250584/1ebac812-da2b-11e5-9a0f-4ad835359508.PNG)

I'm pretty nooby and have no clue what to do. For Help I would be very greatfull, because I couldnt find anything in the web.

I have a Windows 7, 64 bit system. The Wallet is the newest version (0.4.0).


Thanks.
Pipi

**Comment by frozeman** [comment]
Once you switched to the testnet, simply restart the wallet and you should be properly on the testnet.
Type `web3.eth.coinbase` in the console, to see which address is the one you mine into.

Apart from that you simply click the mine button in the menu -> develop -> mine

**Comment by pipilangstrumpf** [comment]
Hi again,
I have restarted my Wallet in the Testnet-mode and as you said, was in the Testnet-mode again. I also checked the mining adress in the console. In both networks (Testnet & Main Network) it is correct.

Unfortunately the mining in the Testnet still doesnt work and the "resource Error" and the "IPC Connection Error" are still there.

Despite from that I tried to get some Ether (7) in the Main Network by buying them via shapeshift. I tried it two times but the transaction couldnt be completed. I think my connection is not right configured somehow. 
The Blockchain is loading. I also checked the firewall, but everything (geth and node) should connect right.

Thank you for your help.

Kind regards,
Pipi
